examples/forms/config/dev.toml

```diff
@@ -0,0 +1,17 @@
+secret_key = "d34f62a5450bed73cb1a5bd750bce5c56c2cad63be59e4e83f7565f33fec30b3"
```

Comment:
Please remove the file and use `ProjectConfig::dev_default()` instead – this slightly reduces the maintenance burden, as we don't have to take care of dozen config files that are essentially the same.